### DISCUSSION TITLE
Need help regarding orbot android 16.6

### DISCUSSION THREAD

**Discussion by Zamhive** [guardianproject/orbot-android#716] [Q&A]
I have connected the vpn but orbot vpn doesn’t allow me to browse or use any app. My data connected but j can't do anything. ![Screenshot_2022-07-20-23-31-17-21_fc704e6b13c4fb26bf5e411f75da84f2.jpg](https://user-images.githubusercontent.com/109689143/180045767-6986947f-bdbe-4d7d-b71c-4e1f7d43dfd3.jpg)

**Comment by RJVoodoo** [comment]
I had a similar issue, I'm not sure how safe this is; but I usually start Orbot; Start Tor once armed; then start the VPN on Orbot. You may also want to check any app resteictions and make sure batter, data are fulll send.
